src/olympia/addons/serializers.py

```diff
@@ -697,25 +699,23 @@ class Meta(BaseUserSerializer.Meta):
 
 
 class AddonAuthorSerializer(serializers.ModelSerializer):
-    user_id = serializers.IntegerField(source='user.id', read_only=True)
```

Comment:
I'm curious about why this was deleted.

Reply:
I realised from tinkering around it wasn't necessary - `user_id` would be serialized into an integer anyway by the `ModelSerializer` code.